plugins/azure/sqldatabases/dbDataMaskingEnabled.js

```diff
@@ -0,0 +1,71 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Set Dynamic Data Masking for SQL Databases',
```

Comment:
```suggestion
    title: 'Data Masking Enabled for SQL Databases',
```
change it to something like this or 'SQL Databases Data Masking Enabled'

Reply:
or SQL Database Dynamic Data Maskin